apps/settings/manifest.webapp

```diff
@@ -44,7 +44,8 @@
         "target": "device"
        },
       "href": "index.html#activity-configure",
-      "disposition": "inline"
+      "disposition": "inline",
+      "returnValue": true
```

Comment:
I see `js/activity_handler.js` has a function `postResult` that could maybe be changed ? It seems to be the only place that calls postResult. It can take a "result" but I see it's always called either without a parameter or with `null`.

So a "right" patch could be to rename this `finishActivity` and change all callers and call window.close instead of activity.postResult. But a simple patch could be to simply add `window.close()` after the `this._currentActivity.postResult` call